contrib/epee/include/storages/http_abstract_invoke.h

```diff
@@ -44,8 +47,11 @@ namespace epee
       if(!serialization::store_t_to_json(out_struct, req_param))
         return false;
 
+      http::fields_list additional_params;
+      additional_params.push_back(std::make_pair("Content-Type","application/json; charset=utf-8"));
+
       const http::http_response_info* pri = NULL;
-      if(!transport.invoke(uri, method, req_param, timeout, std::addressof(pri)))
+      if(!transport.invoke(uri, method, req_param, timeout, std::addressof(pri), additional_params))
```

Comment:
I would consider `std::move`ing `additional_params`. The parameter is taken by const reference, but if it ever changes to by-value then this call will correctly move the contents.